src/main/java/seedu/address/model/person/Location.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's available Location in the TuitionCor.
+ * Guarantees: immutable; is valid as declared in {@link #isValidLocation(String)}
+ */
+public class Location {
+
+    public static final String MESSAGE_LOCATION_CONSTRAINTS =
+            "Location should only be north, south, east, west, central but can take any value if necessary";
```

Comment:
You can also change regex to force the inputs to only take `North`, `South`, etc. Google around to find out more about regex.